jballerina.java.arrays-ballerina/Package.md

```diff
@@ -1,5 +1,15 @@
 ## Package Overview
 
-This package provides APIs to create new Java array instances, get elements from arrays, set elements, etc. 
+The Java arrays library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
+</a> language.
 
-For information on the operations, which you can perform with the jballerina.java.arrays package, see the below **Functions**. 
\ No newline at end of file
+This package provides a set of APIs to do Java array manipulations such as create new Java array instances, get elements from arrays, set elements.
+
+### Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+### Useful Links
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
+- Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
+- Post all technical questions on Stack Overflow with the #ballerina tag.
```

Comment:
Let's add the link here.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays/pull/129/commits/a37d651dc60f2b308a187e568a7d4c4a981550c4